src/r_plane.c

```diff
@@ -434,8 +434,9 @@ static void DrawSkyTex(visplane_t *pl, skytex_t *skytex)
 
         if (dc_yl != USHRT_MAX && dc_yl <= dc_yh)
         {
-            dc_source = R_GetColumnMod2(texture, (an + xtoskyangle[x])
-                                                     >> ANGLETOSKYSHIFT);
+            int angle = (an + xtoskyangle[x]) >> ANGLETOSKYSHIFT;
```

Comment:
Should this rather be an `angle_t`?

Reply:
Maybe we should rename `int angle` to `int col`, it's an argument in the `byte *R_GetColumnMod2(int tex, int col)` function.

Reply:
> My question was if this should be an `unsigned` type, though.

I don't know, it was `signed` previously (argument of `R_GetColumnMod2`).

Reply:
OK.